templates/Dataland/v2/download-card.html.twig

```diff
@@ -28,14 +54,16 @@
                         <strong>File Format: </strong><br>
                         {{ dataset.datasetSubmission.distributionFormatName }}
                     </p>
-                    <hr>
-                    <p class="card-text text-dark">
-                        <a href="#individual-files">Individual Files
-                            <img src="{{ asset('build/images/down-arrow.svg') }}"><br />
-                        </a>
-                    </p>
-                {% endif %}
+                    {% if dataset.availabilityStatus == constant('App\\Entity\\DatasetSubmission::AVAILABILITY_STATUS_PUBLICLY_AVAILABLE') %}
```

Comment:
I don't think this check is necessary as it is the same as line 1

Reply:
It's there in the event the dataset is remotely hosted. We don't want to show the file manager link in that case.

Reply:
I agree, this line is not needed.